docs/Advanced/SQL WHERE Clause.md

```diff
@@ -0,0 +1,45 @@
+---
+publish: true
+---
+
+# WHERE Clause
+<span class="related-pages">#css</span>
+
+## Operators in The WHERE Clause
+
+The following basic operators can be used in the `WHERE` clause. More can be found on the [[5-queries-sql/operators|Operators]] page.
```

Comment:
This is a broken link.

Reply:
The publishing process is Obsidian Publish, so opening `docs/` gives a very good idea of what the published vault will look like, as I set up snippets the same.

(Publish does not support plugins, though)

The `docs/` vault has the Dangling Links plugin enabled, which pointed out this one.

It's a broken link because `5-queries-sql/operators` doesn't exist. Probably it needs updating to a new location.

Reply:
Fixed link.